client/src/lib/urlUtils.ts

```diff
@@ -0,0 +1,27 @@
+import { sanitizeUrl } from "@braintree/sanitize-url";
+import Cookies from "js-cookie";
+
+export const sanitize = (input: string): string => {
+  return sanitizeUrl(input);
+};
```

Comment:
I'm not sure if this function is needed. We can just use sanitizeUrl directly instead :)

Reply:
done